### PR TITLE
docs(EAS Submit): correct link to EAS Build intro

### PR DESCRIPTION
**Pull request by donni106**
- changed the link because it led to the intro of EAS Submit instead of EAS Build as the text says

### CODE REVIEW

**Review by Simek — APPROVED** [review]
Hello @donni106, nice catch, thank you for the correction! 👍 

I have left a small suggestion to convert link to the non-relative form, which should have higher fail possibility in our checks, if the page or content will be moved in the future.

Besides that it LGTM, and PR should be ready to ship after addressing this small link change.

File: docs/pages/submit/ios.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ This guide outlines how to submit your app to the Apple App Store from your own
 
 ## 1. Build a standalone app
 
-You'll need a native app binary signed for store submission. You can either use [EAS Build](introduction.md) or do it on your own.
+You'll need a native app binary signed for store submission. You can either use [EAS Build](../build/introduction.md) or do it on your own.
```
[the remark this answers is not in the document]
Comment (donni106) [reply docs/pages/submit/ios.md:18]: Good to know! I am on it.

[the remark this answers is not in the document]
Comment (donni106) [reply docs/pages/submit/ios.md]: Done, I have decided to keep the `.md` to be consistent with `["Automating submissions" guide](/build/automating-submissions.md).` and `[app config file](/workflow/configuration.md).`

[the remark this answers is not in the document]
Comment (Simek) [reply docs/pages/submit/ios.md]: Just FYI, extension is a legacy thing, which were required to validate the links, but i't no longer needed by the script. We are in the middle of process to clean this up, but we can leave the extension for now. 😉

**Pull request merged**